packages/Popover/stories/Popover.cypress.stories.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
-import windowHandles, { Input } from "@paprika/testing-helpers/lib/windowHandles/";
+import windowHandles, { Input } from "../../testingHelpers/src/windowHandles";
```

Comment:
This is fine since this is only ever executed within the paprika repo, not when it's installed in an app.